### DISCUSSION TITLE
How does Google Map work and what other functions do you think need to be updated on Google Maps?

### DISCUSSION THREAD

**Discussion by proarafat** [community/community#17809] [Other Feature Feedback, Questions, & Ideas]
# How does Google Map work and what other functions do you think need to be updated on Google Maps?

- [x] Research More and Deeply
- [x] Deep Thinking

![Untitled design](https://user-images.githubusercontent.com/96917595/164246749-148a172b-9b88-4733-961e-ef4914172ac4.gif)

**Comment by samus-aran** [comment]
Hi @proarafat ,

Thanks for being a part of the GitHub Community! We're glad you're here. 😄 

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on [Google Maps Community](https://support.google.com/maps/community?hl=en-GB). It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck! 🍀
